clickhouse_db_schema/oss_ci_utilization/oss_ci_time_series_schema.sql

```diff
@@ -0,0 +1,35 @@
+-- This query creates the oss_ci_time_series table on ClickHouse
+CREATE TABLE misc.oss_ci_time_series(
+     -- created_at DateTime when the record is processed in db.
+    `created_at` DateTime64(0,'UTC'),
+    -- type of time series, for instance, utilization log data is 'utilization'.
+    `type` String,
+    `tags` Array(String) DEFAULT [],
+    `time_stamp` DateTime64(0,'UTC'),
+    `repo` String DEFAULT 'pytorch/pytorch',
+    `workflow_id` UInt64,
+    `run_attempt` UInt32,
```

Comment:
this could even be UInt16. 

I would have said UInt8, but one day in the next three years a job will be restarted 256 times and stuff will break :P